crates/common/consensus/src/predicates.rs

```diff
@@ -1,7 +1,17 @@
-use crate::attestation_data::AttestationData;
+use crate::{attestation_data::AttestationData, validator::Validator};
 
 /// Check if ``data_1`` and ``data_2`` are slashable according to Casper FFG rules.
 pub fn is_slashable_attestation_data(data_1: AttestationData, data_2: AttestationData) -> bool {
     (data_1 != data_2 && data_1.target.epoch == data_2.target.epoch)
         || (data_1.source.epoch < data_2.source.epoch && data_2.target.epoch < data_1.target.epoch)
 }
+
+pub fn is_slashable_validator(validator: Validator, epoch: u64) -> bool {
+    if !validator.slashed
+        && validator.activation_epoch <= epoch
+        && epoch < validator.withdrawable_epoch
+    {
+        return true;
+    }
+    false
+}
```

Comment:
```suggestion
pub fn is_slashable_validator(validator: &Validator, epoch: u64) -> bool {
    !validator.slashed
        && validator.activation_epoch <= epoch
        && epoch < validator.withdrawable_epoch
}
```

we don't need the if case for this, and Validator should probably be passed in as a reference, is_slashable_validator probably makes sense to be a method of the Validator Struct, so if you could move this function to `impl Validator` that would be great